dallinger/command_line/docker_ssh.py

```diff
@@ -317,31 +317,30 @@ def deploy(
     executor = Executor(ssh_host, user=ssh_user, app=app_name)
     executor.run("mkdir -p ~/dallinger/caddy.d")
 
-    if not update:
-        print("Removing any pre-existing app with the same name.")
-        app_yml = f"~/dallinger/{app_name}/docker-compose.yml"
-        executor.run(
-            f"if [ -f {app_yml} ]; then docker compose -f {app_yml} down --remove-orphans; fi",
-            raise_=False,
-        )
-        print("Removing any pre-existing Redis volumes.")
-        remove_redis_volumes(app_name, executor)
```

Comment:
I think we still need to leave `remove_redis_volumes` in there as this is not called on `destroy`? By default the database is cleared but I remember that before we had problems if the redis volumes weren't cleared.

Reply:
@silviot would be good to address this when you get a chance